src/interface/src/app/services/auth.service.ts

```diff
@@ -133,19 +133,45 @@ export class AuthService {
       );
   }
 
-  changePassword(password1: string, password2: string): Observable<any> {
+  changePassword(currentPassword: string, newPassword1: string, newPassword2: string): Observable<any> {
     return this.http.post(
       this.API_ROOT.concat('password/change/'),
       {
-        new_password1: password1,
-        new_password2: password2,
+        old_password: currentPassword,
+        new_password1: newPassword1,
+        new_password2: newPassword2,
       },
       {
         withCredentials: true,
       }
     );
   }
 
+  sendPasswordResetEmail(userId: string, email: string): Observable<any> {
+    return this.http.post(
+      this.API_ROOT.concat('password/reset/'),
+      {
+        email: email,
+        userId: userId,
+      },
+      {
+        withCredentials: true,
+      }
+    );
+  }
+
+  verifyToken(token: string): Observable<boolean> {
```

Comment:
Where is this used?

Reply:
Based on the offline sync conversation, there will actually be no need for this method as you mention so will remove for now.